jack-sys/src/lib.rs

```diff
@@ -1102,7 +1113,7 @@ type jack_get_cycle_times_t = unsafe extern "C" fn(
 
 lazy_static! {
     pub static ref jack_get_cycle_times: Option<jack_get_cycle_times_t> = {
-        unsafe { libloading::Library::new(jack_lib) }
+        unsafe { libloading::Library::new(JACK_LIB) }
```

Comment:
Does this work anymore? Anyway, it [isn't needed](https://github.com/RustAudio/rust-jack/issues/159#issuecomment-1023130266), so I suggest getting rid of it.

Reply:
Yes, it's fine to load a library multiple times. `jack_get_cycle_times` is actually not included in the version used by the CI so I'd prefer to keep this.

Reply:
Also: "In the worst case scenario, the library would fail to load with dlib and the application could recover from that."

If any of the functions in a `dlib::external_library!` block are not present, then an error is returned and none of the functions are loaded.

Reply:
Hmm, `jack_get_cycle_times` has been [in JACK1 since 2010](https://github.com/jackaudio/jack1/blob/b04083761496410a52126cdbcd35c557ee82f2e5/libjack/transclient.c#L239)...

Reply:
> jack_get_cycle_times is actually not included in the version used by the CI

I tested moving `jack_get_cycle_times` into one of the `external_library!` macros in [this branch](https://github.com/Be-ing/rust-jack/tree/cycle_time) and [CI worked](https://github.com/Be-ing/rust-jack/actions/runs/1800354092).

Reply:
I'd rather not get sidetracked here so opened #163 to focus on this.

Reply:
okay